proto/aserto/directory/reader/v3/reader.proto

```diff
@@ -226,28 +224,39 @@ message GetObjectsResponse {
     aserto.directory.common.v3.PaginationResponse page          = 9     [(google.api.field_behavior) = OUTPUT_ONLY];
 }
 
-message GetRelationRequest {
+message GetObjectRelationsRequest {
     // object type
     string object_type                                          = 1     [(google.api.field_behavior) = REQUIRED];
     // object identifier
     string object_id                                            = 2     [(google.api.field_behavior) = REQUIRED];
-    // relation name
-    string relation                                             = 3     [(google.api.field_behavior) = REQUIRED];
-    // subject type
-    string subject_type                                         = 4     [(google.api.field_behavior) = REQUIRED];
-    // subject identifier
-    string subject_id                                           = 5     [(google.api.field_behavior) = REQUIRED];
-    // optional subject relation name
+    // diretion of relations to get
+    RelationsTo relations_to                                    = 3     [(google.api.field_behavior) = REQUIRED];
+    // optional filter on object type
+    string neighbor_type                                        = 4     [(google.api.field_behavior) = OPTIONAL];
+    // optional filter on relation type
+    string relation                                             = 5     [(google.api.field_behavior) = OPTIONAL];
+    // optional filter on subject relation name
     string subject_relation                                     = 6     [(google.api.field_behavior) = OPTIONAL];
     // materialize relation objects
-    optional bool with_objects                                  = 7     [(google.api.field_behavior) = OPTIONAL];
+    bool with_objects                                           = 7     [(google.api.field_behavior) = OPTIONAL];
+    // pagination request
+    aserto.directory.common.v3.PaginationRequest page           = 9     [(google.api.field_behavior) = OPTIONAL];
 }
 
-message GetRelationResponse {
+enum RelationsTo {
```

Comment:
Assuming we keep GetObjects which uses incoming and outgoing, we should consolidate the terminology used, instead of adding new verbiage.

Reply:
I think that if we have GetObjectRelations we don't need the WithRelations option in GetObject.
I removed it but forgot to change GetObjectResponse accordingly.

Reply:
GetObject becomes a way to retrieve the properties of a concrete object. Traversing the graph is done with GetRelations and GetObjectRelations.